.gitignore

```diff
@@ -37,4 +37,6 @@ load_data_lambda/vendor/*
 target/
 load_data.zip
 create_async_query.zip
+db_cleanup.zip
```

Comment:
how about just `*.zip`?

Reply:
also: you did check in `db_cleanup_lambda/db_cleanup.zip`. rm this

Reply:
we need to keep the psycopg2.zip file

Reply:
ah ok. You can also `git add -f` that btw.

Reply:
👍